Bottom: 9ff4f5ad21fb2cd91b5e55158f43ac97fb34dffd
Top:    6ee9eed7d73380d08962fe1daf65aac31d427238
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-06 10:16:14 -0500

Refresh of add-buildbot-role-and-new

---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 835ee49c..42db8fdb 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -48,7 +48,7 @@ phpldapadmin_port_args:
 
 phpmyadmin_hostname: phpmyadmin.novatech-llc.com
 
-buildbot_hostname: ldap.novatech-llc.com
+buildbot_hostname: buildbot.novatech-llc.com
 buildbot_port_args:
 - "{{ buildbot_ip_addr }}:80:8080"
 - "{{ buildbot_ip_addr }}:443:8443"
